unison-cli/src/Unison/LSP/Queries.hs

```diff
@@ -360,6 +361,7 @@ annIsFilePosition = \case
   Ann.Intrinsic -> False
   Ann.External -> False
   Ann.Ann {} -> True
+  Ann.GeneratedFrom ann -> annIsFilePosition ann
```

Comment:
An annotation used for generated fields which preserves an annotation about where they came from, but also indicates they're not REALLY in the file.